graphics/Pangolin/Portfile

```diff
@@ -19,9 +19,17 @@ long_description    Pangolin is a lightweight portable rapid development library
                     Pangolin also provides a mechanism for manipulating program variables through config files \
                     and ui integration, and has a flexible real-time plotter for visualising graphical data.
 
-depends_lib-append  port:glew
+depends_build-append \
+                    port:pkgconfig
+
+depends_lib-append  port:glew \
+                    port:ffmpeg \
```

Comment:
This line should be updated to allow ffmpeg-devel to be used also.

Reply:
Done